packages/client/utils/sortByTier.ts

```diff
@@ -4,7 +4,7 @@ const sortByTier = <T extends readonly {tier: TierEnum | string; name: string}[]
   teamsOrOrgs: T
 ) => {
   const teamsSlice = teamsOrOrgs.slice()
-  const tierVal = (team) => (team.tier === 'enterprise' ? -2 : team.tier === 'pro' ? -1 : 1)
+  const tierVal = (team: T[0]) => (team.tier === 'enterprise' ? -2 : team.tier === 'pro' ? -1 : 1)
```

Comment:
#TIL 😁 indexed access types, also called [lookup types](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-2-1.html#keyof-and-lookup-types). 
```ts
type P5 = string[][0]; // string
```

Reply:
heh, never knew they were called that! such a great way to sneak into a typed array